### ISSUE TITLE
Top and skip on expand

### ISSUE THREAD

**Issue by jorgenkarlsson** [object/Simple.OData.Client#248]
Hi,

Is top and skip on expand supported?
I would like to write something like this:

var persons = client
            .For<Person>()
            .Expand(p=> p.Trips.Take(10))
            .FindEntriesAsync().Result;

equals to:
Person?$expand=Trips($top=10)

**Comment by object** [comment]
No, currently top and skip are supported only on a top level, but this is a low hanging fruit that should be easy to implement. Will add it to the list of requested features.

**Comment by jorgenkarlsson** [comment]
Ok, thank you.

**Comment by jmfronsee** [comment]
Hey Vagif sorry to keep bugging you.  I am loving this utility and nice work, but keep bumping into things I can't seem to get right.  I was excited to see this thread on issue #248, but I see it is still open.    My issue is similar to issue #200  "expand with filter" that I asked about earlier.     Basically I have a Parent object (lets keep using my example from other thread) Workstation.   And I want to expand SOME of the WorkStation_Status records on this workstation.  I can see how to Expand ALL, but I really want to only select the latest Top x (sorted in desc order to get the latest).   Tried .For<workstation>().Expand(x => x.workstation_status).Top(10)   and the OP's Take but neither worked  (ok just realized obviously my Top(10) only returns 10 Workstations on the parent level)... Any ideas or other ways I might accomplish this?   THANK YOU AGAIN.
